client/test/unit/specs/transactions.spec.js

```diff
@@ -229,10 +229,10 @@ describe('Testing Transactions.vue', () => {
     expect(vm.$el.querySelector('#transaction-header-names').textContent).toContain('Not good')
 
     expect(vm.$el.querySelector('#transaction-header-info').textContent).toContain('Submitted Date:')
-    expect(vm.$el.querySelector('.submitted-date').textContent).toBe('2021-07-12, 9:49 am Pacific time')
+    expect(vm.$el.querySelector('.submitted-date').textContent).toBe('2021-12-07, 9:49 am Pacific time')
 
     expect(vm.$el.querySelector('#transaction-header-info').textContent).toContain('Expiry Date:')
-    expect(vm.$el.querySelector('.expiry-date').textContent).toBe('2021-09-11, 12:01 am Pacific time')
+    expect(vm.$el.querySelector('.expiry-date').textContent).toBe('2022-02-01, 11:59 pm Pacific time')
```

Comment:
Hmm, 2021-12-07 + 56 days = 2022-01-31. Why this value then?

https://getcalc.com/business-day-calculator.htm

Reply:
Oh sorry! I mis-read that site.

The correct expiry date is indeed Feb 1, 2022.